README.md

```diff
@@ -5,6 +5,15 @@
 ### Requirements
 
 -   [nvm](https://github.com/nvm-sh/nvm)
+-   [mkcert](https://github.com/FiloSottile/mkcert)
+
+#### Install certificates once.
+
+```bash
+mkcert -install
+mkcert -key-file certs/privkey.pem -cert-file certs/cert.pem "*.comet-dxp.dev"
```

Comment:
These files are not included, how is this meant?

Reply:
mkcert creates these files. The naming of the options is rather confusing, as it indicates that these files are used.